tests/python/ci/sample_prs/pr10786-badci.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
are these test cases briefly described anywhere? we could e.g. add a key describing them or to body

Reply:
is there something more than the result that you were looking for? i.e. the actual usages here https://github.com/apache/tvm/pull/10833/files#diff-0dd95b1adca3c8ab9745a9be219a5683885a9710365a447ee290841b172747a6R72-R91

Reply:
just thinking whether we should have `{"testcase_description": "Expected JSON when CI has failed"}` in here

Reply:
added a `detail` to each test case with some expanation